game/state/city/building.h

```diff
@@ -66,6 +66,7 @@ class Building : public StateObject, public std::enable_shared_from_this<Buildin
 	unsigned ticksDetectionTimeOut = 0;
 	unsigned ticksDetectionAttemptAccumulated = 0;
 	bool detected = false;
+	bool investigate = false;
```

Comment:
This probably needs to be added to the serialization code - just adding the member to gamestate_serialize.xml for the building object should be sufficient.

Otherwise saving/loading when an investigation is on it's way will lose the state.